docs/contributing/dev-setup/runtests.rst

```diff
@@ -0,0 +1,16 @@
+.. _runtests:
+
+Run the Tests
+=============
+
+To run the tests provided by a specific plugin, start the django development server in the development vm with `pulp-manager runserver`.
+You need to create a *pulp-smash* config file with `pulp-smash config create` (this is ony needed once).
+Then on a second shell, you can perform the tests via `pulp-manager test <plugin_name>`.
+
+.. note::
+
+    Make sure, the tasks runner are actually running. In doubt, run `prestart`.
```

Comment:
I was referring explicitely to the vagrant development box here. It might be different with pulplift though. I can put `systemctl restart <...>` there if you like.